<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBSettingsPageUi</class>
 <widget class="QWidget" name="OpenRGBSettingsPageUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="CheckboxLoadGeometry">
     <property name="text">
      <string>Load Window Geometry</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="ComboBoxTheme"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="CheckboxMinimizeOnClose">
     <property name="text">
      <string>Minimize On Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ThemeLabel">
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="CheckboxSaveGeometry">
     <property name="text">
      <string>Save Geometry On Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
